spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -263,6 +264,14 @@ def setup(self):
             register_shortcut=True
         )
 
+        goto_cursor_action = self.create_action(
+            DebuggerWidgetActions.GotoCursor,
+            text=_("Show in editor"),
```

Comment:
```suggestion
            text=_("Show in the editor the file and line where the debugger is "
                   "placed"),
```

Make this a bit more verbose so that it's easier to understand what it does.